blockly-svelte/README.md

```diff
@@ -0,0 +1,71 @@
+*Psst — looking for a shareable component template? Go here --> [sveltejs/component-template](https://github.com/sveltejs/component-template)*
```

Comment:
This Readme looks a little generic. Could we just include a link to how to get setup with Svelte here.
And more specific instructions on using this sample?

Similar to the blockly-react sample [README](https://github.com/google/blockly-samples/blob/master/blockly-react/README.md) .